index.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8"/>
+  <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+  <link href='https://fonts.googleapis.com/css?family=Lato' rel='stylesheet'>
+  <link href="https://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"/>
+  <link rel="stylesheet" href="css/mobile-version.css"/>
+  <link rel="stylesheet" href="css/desktop-version.css"/>
+  <script src="https://kit.fontawesome.com/8c5a2b9118.js" crossorigin="anonymous"></script>
+  <title>Youcode</title>
+</head>
+<body>
+  <!-- Main page section -->
+  <section id="main-page">
+    <header>
+      <div class="social-nav">
+        <ul>
+          <li><i class="fa fa-brands fa-facebook-f"></i></li>
+          <li><i class="fa fa-brands fa-twitter"></i></li>
+          <li>English</li>
+          <li>My Page</li>
+          <li>Logout</li>
+        </ul>
+      </div>
+      <div class="navbar">
+        <a class="my-logo" href="#headline">YouCode</a>
+        <ul class="menu">
+          <li><a class="menuItem" href="about.html">About</a></li>
+          <li><a class="menuItem" href="#" style="color: lightgray">Program</a></li>
+          <li><a class="menuItem" href="#" style="color: lightgray">Join</a></li>
+          <li><a class="menuItem" href="#" style="color: lightgray">Sponsor</a></li>
+          <li><a class="menuItem" href="#" style="color: lightgray">News</a></li>
+        </ul>
+        <ul class="desktop-menu">
+          <li><a class="" href="about.html">About</a></li>
+          <li><a class="" href="#"style="color: lightgray">Program</a></li>
+          <li><a class="" href="#" style="color: lightgray">Join</a></li>
+          <li><a class="" href="#" style="color: lightgray">Sponsor</a></li>
+          <li><a class="" href="#" style="color: lightgray">News</a></li>
+          <button class="navbar-btn">CC Campaign</button>
+         </ul>
+        <button class="hamburger">
+          <i class="menuIcon material-icons">menu</i>
+          <i class="closeIcon material-icons">close</i>
+        </button>
+      </div>
+    </header>
+    <main id="headline" class="headline">
+      <p class="greet-title">"Hello! Sharing world!"</p>
+      <h2 class="title">A WEB DEVELOPMENT COURSE TO GET YOU STARTED</h2>
+      <p class="content">
+        Kickstart your courses in web development. Master html, css and javascript
+        courses using hands-on projects involving HTML, JavaScript, React.js, Ruby and more. 
+        No prior experience required.
+      </p>
+      <h2 class="date-set">2023.02.01(MON) - 15(THUR)</h2>
+      <p class="location">(Addis Ababa Arat Kilo Campus IT hall)</p>
```

Comment:
- [ ] Kindly make sure your design stick as much as the possible to the one from the Behance template(e.g.: text color, fonts(size, weigh, family,...), spacing,...)

### Template design
![Screenshot from 2023-02-14 01-18-59](https://user-images.githubusercontent.com/83042742/218596523-bff03fda-6e6b-476d-8334-f9a504f6a418.png)

### Your design
![Screenshot from 2023-02-14 01-19-29](https://user-images.githubusercontent.com/83042742/218596564-c4c41f11-bca9-4dfe-9420-e39430f905e6.png)